tracer/build/_build/_build.csproj

```diff
@@ -5,8 +5,8 @@
     <TargetFramework>net5.0</TargetFramework>
     <RootNamespace></RootNamespace>
     <NoWarn>CS0649;CS0169;SA1652</NoWarn>
-    <NukeRootDirectory>..\..</NukeRootDirectory>
-    <NukeScriptDirectory>..\..</NukeScriptDirectory>
+    <NukeRootDirectory>..\..\..</NukeRootDirectory>
```

Comment:
Maybe these properties should stay the same because the `build.cmd`, `build.ps1`, and `build.sh` scripts were also moved into the `/tracer` directory, but I don't actually know what effect these properties have. Was there an issue that happened when these weren't changed?

Reply:
Just did a quick test

```ps
> mkdir nuke-test
> cd nuke-test
> dotnet new sln
> nuke :setup
How should the build project be named?
¬  _build
Where should the build project be located?
¬  ./tracer/build/_build
Which NUKE version should be used?
¬  5.3.0 (latest release)
Which solution should be the default?
¬  repos.sln
Do you need help getting started with a basic build?
¬  No, I can do this myself...
```

and tracer/build/_build/_build.csproj contains

```xml
    <NukeRootDirectory>..\..\..</NukeRootDirectory>
    <NukeScriptDirectory>..\..\..</NukeScriptDirectory>
```

So I think this is correct!

Reply:
Thanks for confirming that Andrew! Disregard my comment 😄

Reply:
oh, actually, the build.cmd and build.ps1 were generated in the root directory, so maybe `NukeScriptDirectory` _should_ be as it was 🤔

Reply:
Maybe we can revert the Nuke changes for now and see what happens

Reply:
yes, by leaving `NukeScriptDirectory` I get all the build scripts back in the solution. But the NukeRootDirectory seems good with 3 levels on top, All the scripts with the red line reappear 
![image](https://user-images.githubusercontent.com/8353486/134674395-b352f702-83f3-44ba-9f9f-16ed491b85cd.png)